core/modules/deserializers.js

```diff
@@ -142,7 +91,19 @@ function deserializeHtmlFile(text,fields) {
 	return [result];
 }
 
-function deserializeTiddlyWikiFile(text,storeAreaEnd,isTiddlyWiki5,fields) {
+function deserializeNewStoreArea(text,storeAreaEnd,type,fields) {
```

Comment:
Would something along the lines of `deserializeJSONStoreArea` be a more appropriate name? 

I am just postulating a future in which we introduce yet another store area format and then "NewStoreArea" becomes ambiguous.

Reply:
From the perspective of the deserializer, none of this is JSON-specific: we're adding support for embedding blocks for any supported deserializers.

I thought of "scriptStoreArea", but the core already supports tiddlers in script tags in a slightly different format for modules.